api/routes.py

```diff
@@ -570,14 +598,16 @@ def saml_callback():
     return app.manager.saml_controller.saml_authentication_callback(request, app.manager._db)
 
 
-@app.route('/lcp/hint')
+@library_route('/lcp/hint')
+@has_library
```

Comment:
Presumably you're going to change this later to use `has_library_through_external_loan_identifier`. If you're confident that your LCP branch will be accepted, I think the change can happen now; we already know the code won't work as is.

Reply:
I updated the `hint` route to `/<collection_name>/lcp/licenses/<license_id>/hint'`